docs/index.html

```diff
@@ -45,7 +45,12 @@ <h3>Get up and running <em>in&nbsp;seconds</em>.</h3>
         <p class="line">
           <span class="path">~</span>
           <span class="prompt">$</span>
-          <span class="command">gem install jekyll bundler</span>
+          <span class="command">gem install bundler</span>
+        </p>
+        <p class="line">
+          <span class="path">~</span>
+          <span class="prompt">$</span>
+          <span class="command">bundle add jekyll && bundle install</span>
```

Comment:
Should we use `&amp;` instead here?

Reply:
@crispgm, I used it because it is the convention in Bash to indicate “Run this command, now run that command” after the first one completes. I don’t know enough with regard to character encodings to say if this would cause Unicode issues and not display correctly in some browsers.

Reply:
Tried it out just to be sure, looks the same, so I'm 👍🏼 on this.